### PR TITLE
Wb missing alttext 14

### PR DESCRIPTION
**Pull request by MeeraDi**

### CODE REVIEW

File: iis/manage/managing-your-configuration-settings/editing-collections-using-configuration-editor-complex-sections.md [hunk 1]
```diff
@@ -27,30 +27,30 @@ This walkthrough requires the following prerequisites:
 2. Select the Server in the Connections Tree View
 3. Open the Configuration Editor User Interface module
 
-    [![](editing-collections-using-configuration-editor-complex-sections/_static/image2.jpg)](editing-collections-using-configuration-editor-complex-sections/_static/image1.jpg)
+    ![Screensheet of the I I S Manager window. The server home features is in the main pane.](editing-collections-using-configuration-editor-complex-sections/_static/image1.jpg)
```
Comment (v-emilypr) [inline iis/manage/managing-your-configuration-settings/editing-collections-using-configuration-editor-complex-sections.md:30]: Screenshot

File: iis/troubleshoot/performance-issues/troubleshooting-iis-performance-issues-or-application-errors-using-logparser.md [hunk 2]
```diff
@@ -45,12 +45,12 @@ By default, IIS log files are located in the following directories:
 
 In this troubleshooter, I will be using IIS 8. Open the **IIS Manager** and select **Sites**, as shown in *Figure 1*. This will show you the ID of each website hosted on your server. You will need this ID to determine which W3SVC\* directory to analyze.
 
-![](troubleshooting-iis-performance-issues-or-application-errors-using-logparser/_static/image1.png)  
+![Screenshot of I I S Manager window showing Sites in the main pane.](troubleshooting-iis-performance-issues-or-application-errors-using-logparser/_static/image1.png)
```
Comment (v-emilypr) [inline iis/troubleshoot/performance-issues/troubleshooting-iis-performance-issues-or-application-errors-using-logparser.md:48]: Screenshot of **the** I I S Manager...
for consistency?

**Comment by MeeraDi** [comment]
@Rick-Anderson @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. 
Thank you!

File: iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/directoryBrowse.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Default FTP Directory Browse Settings &lt;directoryBrowse&gt;"
 author: rick-anderson
-description: "Overview The &lt;directoryBrowse&gt; element is used to specify the content settings for directory browsing on FTP sites. When you configure directory browsi..."
+description: The `<directoryBrowse>` element is used to specify the content settings for directory browsing on FTP sites.
```
Comment (Rick-Anderson) [inline iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/directoryBrowse.md:4]: Nice fix. Hopefully you can get the others to fix line 4, `description` warnings. Thanks

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**